### ISSUE TITLE
After upgrading to 3.3.5 all requests on API Gateway now return base64 instead of JSON

### ISSUE THREAD

**Issue by Soabirw** [awslabs/aws-serverless-express#200]
We were on version ^3.1.3 and without changes on our end the next deploy to our dev stage took down entire API by having each return raw base64 encoded body instead of JSON like usual.

Locking npm into version 3.1.3 resolved this issue.  Locally, 3.3.5 works just fine.  But once deployed to API Gateway then all responses are base64.

Is there changes we need to be aware of for this migration?  Do we need to add JSON as a binary media type?

**Comment by brettstack** [comment]
Could you supply repro steps? This would be a bug.

**Comment by brettstack** [comment]
If it's a bug, it's likely here https://github.com/awslabs/aws-serverless-express/blob/master/src/index.js#L38

**Comment by Soabirw** [comment]
I can try and put a code sample together.  It took a while to narrow this down.  We compared the package-lock.json against a known good Lambda version and this is one that stood out.  It looked like there were notable changes between the versions.

Locking in 3.1.3 fixed the issue.  Updating to 3.3.5 and deploying again re-broke the response bodies.

However, as I went to provide some code samples I noticed in our handler that we had this line:

`const server = awsServerlessExpress.createServer(app, null, ['image/png', 'image/jpeg', 'image/webp', 'image/apng', 'image/*', '*/*']);`

This is back when our API tried to also serve images (which worked except in FF).  We moved that to an entirely new API that only served binary data as mixing binary and JSON was proving to be too difficult for all use cases.

This must have been ignored for JSON until recent changes?  So it makes sense with the '*/*' in there that JSON would now start to be encoded.

I remove that configuration on our non-image API and it now sends back JSON correctly even on latest version.
